wso2is_analytics/hieradata/dev/wso2/wso2is_analytics/pattern-1/README.md

```diff
@@ -0,0 +1,168 @@
+# WSO2 Identity Analytics Server Pattern-1
+
+[WSO2 Identity Server Deployment Pattern 2](https://docs.wso2.com/display/IS540/Deployment+Patterns#DeploymentPatterns-Pattern2-HAclustereddeploymentofWSO2IdentityServerwithWSO2IdentityAnalytics) 
+runs two Identity Server instances fronted with a load balancer with WSO2 
+Identity Analytics Server. As WSO2 Identity Analytics Server component is not mission critical, a single node 
+deployment of a WSO2 Identity Analytics Server or the [minimum HA deployment of WSO2 Identity Analytics Server](https://docs.wso2.com/display/IS540/Setting+Up+Deployment+Pattern+2#SettingUpDeploymentPattern2-MinimumHighAvailabilityDeploymentforWSO2ISAnalytics) 
+can be done.
+
+![alt tag](../../../../../../patterns/images/deployment-architecture-pattern-2.png)
+  
+The hiera YAML in this directory can be used to setup WSO2 Identity Analytics Server nodes participating in the 
+above deployment.
+Default configurations in the hiera YAML is to setup a single node deployment of WSO2 Identity Analytics Server.
+
+Follow the given steps below, to configure hiera YAML with respect to the deployment.
+
+1. Update hostname
+
+    Update ```wso2::hostname``` and ```wso2::mgt_hostname``` in the [default.yaml](default.yaml) file with the DNS name 
+    of the load balancer. If the environment does not have a load balancer, then make sure to add the hostname mapping to 
+    /etc/hosts file. 
+    e.g:
+    ```
+    wso2::hostname: is.analytics.dev.wso2.org
+    wso2::mgt_hostname: is.analytics.dev.wso2.org
+    ```
+2. Configure JDBC driver jar file name
+
+    Configure the JDBC driver file name under ```wso2::file_list``` or in common.yaml. For MYSQL there is an 
+    entry in common.yaml as ```wso2::datasources::mysql::connector_jar``` which is then referred by the [default.yaml]
+    (default.yaml) file here.
+    e.g:
+     ```
+     wso2::file_list:
+       - "repository/components/lib/%{hiera('wso2::datasources::mysql::connector_jar')}"
+    ```
+3. Configure datasources
+
+    Update the data sources to point to an external RDBMS server. 
+    
+    You can configure the database server engine, host name and port under below hiera configurations.
+    These properties are then used in configuring connection URLs in hiera.
+    ```
+    wso2::rdbms::engine: mysql
+    wso2::rdbms::hostname: mysql.sever.com
+    wso2::rdbms::port: 3306
+    ```
+
+    For each datasource you will have to configure the connection URL against ```url``` adding the database name and 
+    other connection URL query parameters, connection username against  ```username``` and password against 
+    ```password``` property keys. Datasources defined in this hiera file are below.
+    * Datasource in repository/conf/master-datasources.xml file:
+    
+    A single datasource can be configured for registry and user management
+    e.g:
+    ```
+    wso2::master_datasources:
+      analytics_ds:
+          name: WSO2_ANALYTICS_DS
```

Comment:
How can this be an example for registry and user management ?

Reply:
'datasource' element under 'wso2::user_management' and 'wso2::registry_instances' points to the datasource defined here, which are used in The registry.xml and user-mgt.xml templates